sdk/hybridaks/Azure.ResourceManager.HybridContainerService/src/Generated/VirtualNetworkResource.cs

```diff
@@ -0,0 +1,514 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager;
+using Azure.ResourceManager.HybridContainerService.Models;
+using Azure.ResourceManager.Resources;
+
+namespace Azure.ResourceManager.HybridContainerService
+{
+    /// <summary>
+    /// A Class representing a VirtualNetwork along with the instance operations that can be performed on it.
+    /// If you have a <see cref="ResourceIdentifier" /> you can construct a <see cref="VirtualNetworkResource" />
+    /// from an instance of <see cref="ArmClient" /> using the GetVirtualNetworkResource method.
+    /// Otherwise you can get one from its parent resource <see cref="ResourceGroupResource" /> using the GetVirtualNetwork method.
+    /// </summary>
+    public partial class VirtualNetworkResource : ArmResource
```

Comment:
`VirtualNetwork` is too generic. Please add a prefix to the resource name to make it more descriptive.

Reply:
Thanks for the suggestion, I've renamed it now